projects/kalidea/kaligraphi/src/lib/atoms/kal-snackbar/kal-snackbar.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { OverlayModule } from '@angular/cdk/overlay';
+import { PortalModule } from '@angular/cdk/portal';
+
+import { KalSnackbarComponent } from './kal-snackbar.component';
+
+export * from './kal-snackbar-config';
+export * from './kal-snackbar.injector';
+export * from './kal-snackbar.component';
+export * from './kal-snackbar.service';
+
+const exports = [
+  KalSnackbarComponent
+];
+
+@NgModule({
+  declarations: exports,
+  exports: [...exports, OverlayModule, PortalModule],
```

Comment:
Format code

Reply:
what do you mean by "format code" ?

Reply:
`    exports: [
      ...exports,
      OverlayModule,
      PortalModule
    ];`

Reply:
you have to break lines between variables in exports array